packages/hash/frontend/src/blocks/page/Comments/CommentBlock.tsx

```diff
@@ -130,13 +176,61 @@ export const CommentBlock: FunctionComponent<CommentProps> = ({ comment }) => {
       </Box>
 
       <Box p={0.5} pt={2} position="relative">
-        <CommentTextField
-          onLineCountChange={(lines) => setShouldCollapse(lines > 2)}
-          value={hasText}
-          className={collapsed ? styles.Comment__TextField_collapsed! : ""}
-          readOnly
-        />
-        {shouldCollapse && collapsed ? (
+        <Box
+          sx={({ palette, transitions }) => ({
+            border: `${editable ? 1 : 0}px solid ${palette.gray[30]}`,
+            pl: editable ? 2 : 0,
+            borderRadius: 1.5,
+            transition: transitions.create(["padding", "border-color"]),
+            "&:focus-within": {
+              borderColor: palette.blue[60],
+            },
+          })}
+        >
+          <CommentTextField
+            onLineCountChange={(lines) => setShouldCollapse(lines > 2)}
```

Comment:
Requested change: line count is also calculated when the text is in editing mode, which means that if the text in the input is 3 lines, but the text when saved is 2 lines, 'Show more' appears briefly then disappears after saving the comment. Can we fix this please?

Reply:
Nice spot, thanks! Fixed in 0f515b5dc3a86d461a6d4884fe3bf1f4e3af9741.